src/components/Board/TileEditor/TileEditor.component.js

```diff
@@ -343,6 +482,29 @@ export class TileEditor extends Component {
                         <Symbol image={tileInView.image} label={currentLabel} />
                       </Tile>
                     </div>
+                    {this.state.isEditImageBtnActive && (
+                      <React.Fragment>
+                        <ImageEditor
+                          intl={intl}
+                          open={this.state.openImageEditor}
+                          onImageEditorClose={this.onImageEditorClose}
+                          onImageEditorDone={this.onImageEditorDone}
+                          image={URL.createObjectURL(
+                            this.state.imageUploadedData[this.state.activeStep]
+                              .blobHQ
+                          )}
+                        />
+                        <Button
+                          variant="contained"
+                          color="secondary"
+                          startIcon={<EditIcon />}
+                          onClick={this.handleOnClickImageEditor}
+                          className="TileEditor__EditImage"
```

Comment:
you can add a property 'style' to Button component with `{ marginBottom: '6px' }` to avoid the use of !important.

Reply:
Nice!